opensrp-giz-malawi/src/main/java/org/smartregister/giz/fragment/GizMeFragment.java

```diff
@@ -0,0 +1,66 @@
+package org.smartregister.giz.fragment;
+
+import android.app.Activity;
+import android.view.View;
+
+import org.smartregister.anc.library.fragment.MeFragment;
+import org.smartregister.giz.R;
+import org.smartregister.giz.activity.AncRegisterActivity;
+import org.smartregister.giz.activity.ChildRegisterActivity;
+import org.smartregister.giz.activity.OpdRegisterActivity;
+import org.smartregister.giz.application.GizMalawiApplication;
+import org.smartregister.giz.listener.OnLocationChangeListener;
+import org.smartregister.giz.util.GizUtils;
+import org.smartregister.view.LocationPickerView;
+
+public class GizMeFragment extends MeFragment implements OnLocationChangeListener {
```

Comment:
I am having some difficulty understanding why this is here and will all the code below

Reply:
We talked about this